lib/components/artist/artworks/artwork.js

```diff
@@ -1,83 +0,0 @@
-/* @flow */
```

Comment:
so this is the `artist /artworks/artwork.js` which is an unused dupe of `artwork_grids/artwork.js`